microscopium/screens/cellomics.py

```diff
@@ -29,8 +30,12 @@ def batch_stitch_stack(file_dict, output, order=[0, 1, 2], target_bit_depth=8, *
         files. This dictionary is built using the ``make_key2file`` function.
     output : string
         The directory to output the stitched and concatenated images to.
-    order : list of int, optional
+    stitch_order : array of int, shape (M, N)
+        The order of the stitching.
+        Passed to ``microscopium.cellomics.snail_stitch``
```

Comment:
The documentation seems to suggest that the ``:py:func:`` tag should be added to function references too?